src/tlo/methods/healthsystem.py

```diff
@@ -787,6 +795,7 @@ def on_simulation_end(self):
         if self._hsi_event_count_log_period == "simulation":
             self._write_hsi_event_counts_to_log_and_reset()
             self._write_never_ran_hsi_event_counts_to_log_and_reset()
+            self._write_hsi_event_no_blank_counts_to_log_and_reset()
```

Comment:
no needed (I don't think) as these are to do with the full logger, that captrues the exact footprint anyway.